sqlparse/filters.py

```diff
@@ -737,6 +741,18 @@ def _create_column_type_length(self, token_queue):
                 tokens=parenthesis_token.tokens
             )
 
+    def _create_column_type_values(self, token_queue):
+        if len(token_queue) <= 0:
+            return None
+        if isinstance(token_queue[0], sql.Parenthesis):
+            parenthesis_token = token_queue.popleft()
+            return sql.ColumnTypeValues(
+                tokens=[sql.Token(
+                    value=token.value.strip('`"\''),
```

Comment:
Thanks for spotting this @clin7, i fixed this by `_clean_string_quote` in `filters.py`